README.md

```diff
@@ -139,10 +139,8 @@ In addition to code blocks, some code blocks should be rendered differently depe
 ### Example
 
 <pre>
-```
 $ git init
 Initialized empty Git repository in /Users/skills/Projects/recipe-repository/.git/
-```
 </pre>
```

Comment:
This suggestion means to revert the deleted line though it seems to delete </pre> when rendered.
```suggestion
```
</pre>
```